src/views/templates/list/hooks/useWorkloadProfile.ts

```diff
@@ -1,23 +1,24 @@
+import { TFunction } from 'react-i18next';
+
 import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
-import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 
 import { getTemplateWorkload } from '../../utils/selectors';
 
-// t('plugin__kubevirt-plugin~Desktop')
-// t('plugin__kubevirt-plugin~Server')
-// t('plugin__kubevirt-plugin~High performance')
-// t('plugin__kubevirt-plugin~Other')
+// t('Desktop')
+// t('Server')
+// t('High performance')
 
-const workloadLabels = {
+export const workloadProfiles = {
   desktop: 'Desktop',
   server: 'Server',
   highperformance: 'High performance',
 };
 
-const useWorkloadProfile = (template: V1Template): string => {
-  const { t } = useKubevirtTranslation();
-
-  return t(workloadLabels[getTemplateWorkload(template)] || 'Other');
-};
+export type WorkloadProfileNames = 'Desktop' | 'Server' | 'High performance';
```

Comment:
```suggestion
// t('plugin__kubevirt-plugin~Desktop')
// t('plugin__kubevirt-plugin~Server')
// t('plugin__kubevirt-plugin~High performance')
// t('plugin__kubevirt-plugin~Other')
export type WorkloadProfileNames = 'Desktop' | 'Server' | 'High performance' | 'Other';
```

Reply:
We don't need the prefix `plugin__kubevirt-plugin~` because its already the default namespace.

Regarding 'Other', I would better avoid adding 'Other' to workload profiles, as 'Other' is not a workload profile, but an exception, edge case. Its purpose is to display 'Other' in the templates list or Details tab for the situations when there's other than Desktop, Server, High performance set as template's workload profile.